src/icons/states/UploadEmptyState.js

```diff
@@ -18,31 +19,71 @@ const UploadEmptyState = ({ className = '', color = BOX_BLUE, height = 55, title
18,119 C128.778398,119.151814 128.651814,119.278398 128.5,119.366218 L128.5,120.009521 C128.5,120.280405 128.268066,120.5 128,120.5 C127.723858,120.5 127.5,120.268979 127.5,119.995256 L127.5,119.366218 C127.348186,119.278398 127.221602,119.151814 127.133782,119 L126.490479,119 C126.219595,119 126,118.768066 126,118.5 C126,118.223858 126.215057,118 126.490479,118 L127.133782,118 C127.221602,117.848186 127.348186,117.721602 127.5,117.633782 Z M28.9375,33.7086139 L28.9375,33.3073672 C28.9375,33.137613 29.0824585,33 29.25,33 C29.422589,33 29.5625,33.1344109 29.5625,33.3065491 L29.5625,33.7086139 C29.6573838,33.7635012 29.7364988,33.8426162 29.7913861,33.9375 L30.1934509,33.9375 C30.3627533,33.9375 30.5,34.0824585 30.5,34.25 C30.5,34.422589 30.3655891,34.5625 30.1934509,34.5625 L29.7913861,34.5625 C29.7364988,34.6573838 29.6573838,34.7364988 29.5625,34.7913861 L29.5625,35.1934509 C29.5625,35.3627533 29.4175415,35.5 29.25,35.5 C29.077411,35.5 28.9375,35.355612 28.9375,35.1974433 L28.9375,34.7913861 C28.8426162,34.7364988 28.7635012,34.6573838 28.7086139,34.5625 L28.3065491,34.5625 C28.1372467,34.5625 28,34.4175415 28,34.25 C28,34.077411 28.1344109,33.9375 28.3065491,33.9375 L28.7086139,33.9375 C28.7635012,33.8426162 28.8426162,33.7635012 28.9375,33.7086139 Z M11.4375,102.708614 L11.4375,102.307367 C11.4375,102.137613 11.5824585,102 11.75,102 C11.922589,102 12.0625,102.134411 12.0625,102.306549 L12.0625,102.708614 C12.1573838,102.763501 12.2364988,102.842616 12.2913861,102.9375 L12.6934509,102.9375 C12.8627533,102.9375 13,103.082458 13,103.25 C13,103.422589 12.8655891,103.5625 12.6934509,103.5625 L12.2913861,103.5625 C12.2364988,103.657384 12.1573838,103.736499 12.0625,103.791386 L12.0625,104.193451 C12.0625,104.362753 11.9175415,104.5 11.75,104.5 C11.577411,104.5 11.4375,104.355612 11.4375,104.197443 L11.4375,103.791386 C11.3426162,103.736499 11.2635012,103.657384 11.2086139,103.5625 L10.8065491,103.5625 C10.6372467,103.5625 10.5,103.417542 10.5,103.25 C10.5,103.077411 10.6344109,102.9375 10.8065491,102.9375 L11.2086139,102.9375 C11.2635012,102.842616 11.3426162,102.763501 11.4375,102.708614 Z M118.4375,113.208614 L118.4375,112.807367 C118.4375,112.637613 118.582458,112.5 118.75,112.5 C118.922589,112.5 119.0625,112.634411 119.0625,112.806549 L119.0625,113.208614 C119.157384,113.263501 119.236499,113.342616 119.291386,113.4375 L119.693451,113.4375 C119.862753,113.4375 120,113.582458 120,113.75 C120,113.922589 119.865589,114.0625 119.693451,114.0625 L119.291386,114.0625 C119.236499,114.157384 119.157384,114.236499 119.0625,114.291386 L119.0625,114.693451 C119.0625,114.862753 118.917542,115 118.75,115 C118.577411,115 118.4375,114.855612 118.4375,114.697443 L118.4375,114.291386 C118.342616,114.236499 118.263501,114.157384 118.208614,114.0625 L117.806549,114.0625 C117.637247,114.0625 117.5,113.917542 117.5,113.75 C117.5,113.577411 117.634411,113.4375 117.806549,113.4375 L118.208614,113.4375 C118.263501,113.342616 118.342616,113.263501 118.4375,113.208614 Z"
```

Comment:
Can you try running this through an svg optimizer? It looks a lot larger than the old one.

Reply:
I did run in through an optimizer, I can certainly try another one tho!